Cargo.toml

```diff
@@ -12,6 +12,8 @@ protox = "0.7.1"
 tonic-build = "0.12.3"
 
 [dependencies]
+bdk = { version = "0.29.0", features = ["all-keys", "sqlite"] }
+rusqlite = "0.28.0"
```

Comment:
Please try to keep dependencies in lexicographic order.
We could use https://github.com/DevinR528/cargo-sort to enforce this, but I think it's not worth it for a single lint